slack/Module.md

```diff
@@ -0,0 +1,67 @@
+## Overview
+This module allows you to access the Slack Web API through Ballerina. This module provides the capability
+to query information from and perform some actions in a Slack workspace.
+
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+
+* Create a Slack account.
+    1. Visit https://slack.com/get-started#/createnew and create a Slack account.
+* Obtain Slack User Oauth Token.
+    1. Visit https://api.slack.com/apps and create a Slack App.
+    2. In the `Add features and functionality` section, Click `Permissions`.
+    3. Go to the `Scopes` section and add necessary OAuth scopes in `User Token Scopes` section. ("channels:history", "channels:read", "channels:write", "chat:write", "emoji:read", files:read", "files:write", "groups:read", "reactions:read", "users:read", "users:read.email")
+    4. Go back to `Basic Information` section of your Slack App, go to `Install your app` section and Install the app to the workspace by clicking `Install to Workspace` button.
```

Comment:
Go back to `Basic Information` section of your Slack App. Then go to `Install your app` section and install the app to the workspace by clicking `Install to Workspace` button.

Reply:
done